ui-admin/src/forms/designer/HtmlDesigner.tsx

```diff
@@ -3,20 +3,31 @@ import React from 'react'
 import { HtmlElement } from '@juniper/ui-core'
 
 import { Textarea } from 'components/forms/Textarea'
+import { Button } from '../../components/forms/Button'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
 
 export type HtmlDesignerProps = {
   element: HtmlElement
   readOnly: boolean
   onChange: (newValue: HtmlElement) => void
+  addNextQuestion: () => void
 }
 
 /** UI for editing an HTML element in a form. */
 export const HtmlDesigner = (props: HtmlDesignerProps) => {
-  const { element, readOnly, onChange } = props
+  const { element, readOnly, onChange, addNextQuestion } = props
 
   return (
     <div className="d-flex flex-column h-100">
-      <h2>{element.name}</h2>
+      <div className="d-flex align-items-center justify-content-between">
+        <h2>{element.name} (html)</h2>
+        {addNextQuestion && <div>
+          <Button variant="secondary" className="ms-auto" onClick={addNextQuestion}>
+            <FontAwesomeIcon icon={faPlus}/> Add question
```

Comment:
It's unclear why this is not 'Add new question'. Is it because HTML is not technically a question? Will users see it that way? (We include 'Html' as a question type in the new question dropdown.) 
IOW, might this be confusing? (Am I adding a question to this HTML element?)

Reply:
good catch, this should be "Add next question"